pkg/plugin/aggregation/aggregator.go

```diff
@@ -157,13 +163,13 @@ func (a *Aggregator) isComplete() bool {
 	return true
 }
 
-func (a *Aggregator) isResultExpected(result *plugin.Result) bool {
-	_, ok := a.ExpectedResults[result.ExpectedResultID()]
+func (a *Aggregator) isResultExpected(obj keyer) bool {
```

Comment:
Should rename this to just `isExpected` since now it applies to status updates as well.